packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx

```diff
@@ -142,10 +142,10 @@ const ViewerHome: React.FC = () => {
           new MeasureToolsUiItemsProvider(),
         ]}
         extensions={[
-          new LocalExtensionProvider({
-            manifestPromise: LocalExtension.manifestPromise,
-            main: LocalExtension.main,
-          }),
+          // new LocalExtensionProvider({
```

Comment:
why is this commented?

Reply:
had run into this earlier, had to clean and rebuild a few times before it worked. 
reverted.

<img width="595" alt="image" src="https://user-images.githubusercontent.com/11051042/179826570-ecc42f59-2c02-4e05-9a98-39d722b6aeaf.png">